src/Answers.tsx

```diff
@@ -75,20 +72,6 @@ const AnswerWithSVG = ({
         >
             {isCorrect ? <SvgCheckmark /> : <SvgCross />}
         </div>
-        <input
```

Comment:
There were some issues of loosing focue for keyboard users and we do not need input once a selection has been made.